### PR TITLE
fix: Fix code highlight bug

### PR DESCRIPTION
**Pull request by ErikQQY**
codehighlight

Signed-off-by: ErikQQY <2283984853@qq.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@svgr/webpack": "^5.5.0",
     "clsx": "^1.1.1",
     "file-loader": "^6.2.0",
+    "prism-react-renderer": "^1.2.1",
```
Comment (hsluoyz) [inline package.json:25]: yarn.lock?

**Review by hsluoyz — CHANGES_REQUESTED** [review]
And what's the difference?

File: package.json (hunk 1 above)
Comment (ErikQQY) [reply package.json:25]: The original code theme can't highlight some key words and can't change along with dark mode.

after



original

Comment (ErikQQY) [reply package.json:25]: The prism-react-renderer package is added by docusaurus in yarn.lock: https://github.com/casdoor/casdoor-website/blob/master/yarn.lock#L6700

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**